docs/06_how-to-guides/60_how-to-return-values-from-actions.md

```diff
@@ -2,19 +2,21 @@
 content_title: How-To Return Values From Action
 ---
 
-## Summary
+## Overview
 
 This how-to demonstrates how a smart contract developer implements return values from an action.
 
 In order to accomplish this, use the `return` statement and pass the desired returned value, which can be of any C++ primitive type, a standard library type, or a user defined type.
 
-## Prerequisites
+## Before you begin
+
+Complete the following prerequisites:
```

Comment:
perhaps "This how to requires:"

Complete the following prerequisites : An EOSIO development env. etc ...  sounds funny

Reply:
hmmm, our template has this exact format 

"## Before you begin

Complete the following prerequisites: ....."

if you believe it is wrong let's bring it up with @sunshineveg

Reply:
I think you have missed the point here:

What you have written equates to:

 Complete the following prerequisites. An EOSIO development environment, for details consult the [Get Started Guide](https://developers.eos.io/welcome/latest/getting-started-guide/index).
Complete the following prerequisites. A smart contract, let’s call it `smrtcontract`, which builds without error.
Complete the following prerequisites. An action, let’s call it `checkwithrv`, from which you want to return a value of a user defined type `action_response`.

If you read them as a sentence they do not 'flow'. I was suggesting that changing  "Complete the following prerequisites" may improve the 'flow' and be easier to change than each of the prerequisites. But if  consensus is that this reads correctly then thats ok. It was a suggestion.

Reply:
changed to
`Make sure you have the following prerequisites in place:`
followed by nouns not actions.